Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
```

Comment:
And I guess when @evenator made this PR there might have not been ROS kinetic image yet.